pool/web/view/page/page_admin_mailings.ml

```diff
@@ -247,14 +247,21 @@ let form
       CCOption.map_or
         ~default:false
         (fun dist ->
+          let dist =
+            match dist with
```

Comment:
how about "find_dist" method for the dist module?

Reply:
I don't see where else this could be used. In the other cases, the pattern match returns an option or a bool.

Reply:
https://github.com/uzh/pool/pull/59/files#diff-74630ce402e1242914ef3cafed8fd4632bd51e597405f6a05bae0d053e72544eR398

Reply:
I see, but this is an `option` of dist, so we'd have to wrap in in another match.

But I can create such a function.